src/Uno.Extensions.Validation/ValidationBuilder.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.Extensions.Configuration;
+
+namespace Uno.Extensions.Validation;
+
+internal record ValidationBuilder(IHostBuilder HostBuilder) : IValidationBuilder
+{
```

Comment:
This is similar to ConfigBuilder, so perhaps we should extract the methods to BuilderBase (this would be in hosting library) - both ValidationBuilder and ConfigBuilder would inherit from BuilderBase. Before you do this, just check to make sure we don't already have a BuilderBase defined in extensions

Reply:
I think we already have a BaseBuilder.cs defined:


```
public record BaseBuilder : IBuilder
{
	public IHostBuilder? HostBuilder { get; init; }
}

public record BaseBuilder<T> : BaseBuilder, IBuilder<T>
	where T : new()
{
	public T Settings { get; set; } = new T();
}
```